website/docs/r/load_balancer_monitor.html.markdown

```diff
@@ -10,6 +10,9 @@ description: |-
 
 If you're using Cloudflare's Load Balancing to load-balance across multiple origin servers or data centers, you configure one of these Monitors to actively check the availability of those servers over HTTP(S) or TCP.
 
+*Note*
+When creating a monitor, you have to pass `account_id` to the provider configuration.
```

Comment:
There is some special Terraform website markup we can use here to make this a callout

```suggestion
-> **Note:** When creating a monitor, you have to pass `account_id` to the provider configuration.
```

See https://registry.terraform.io/providers/cloudflare/cloudflare/latest/docs/resources/access_application for an example of it in action.